### ISSUE TITLE
Changes not saved to build

### ISSUE THREAD

**Issue by kubukoz** [scalacenter/scala3-migrate#178]
Hi! First of all, great work on the tool - I'm sure it'll be greatly appreciated by the community.

I tried migrating a minimal project: https://github.com/kubukoz/daniels-regex-interpreter/tree/bd3da8f3efe6058639df615c87cd74043cbd5b1f - I ran `migrate root` after adding the sbt plugin and bumping sbt, and there were no changes to files (not even the build.sbt file).

Is that intended, or should the build definition be updated as well and this is a bug?

My setup:

```
//build.properties
sbt.version=1.5.0-RC2

//project/plugins.sbt
addSbtPlugin("ch.epfl.scala" % "sbt-scala3-migrate" % "0.3.1")
```

Original build.sbt file:

```scala
ThisBuild / organization := "com.example"
ThisBuild / scalaVersion := "2.13.4"

lazy val root = (project in file(".")).settings(
  name := "ce3.g8",
  libraryDependencies ++= Seq(
    "co.fs2" %% "fs2-core" % "3.0.0-M9"
  )
)
```

Sources: irrelevant (no changes were required to compile under 3.0.0-RC1)

Output from `migrate root`:

```
[success] Total time: 0 s, completed 24 Mar 2021, 17:03:17
[info] compiling 1 Scala source to /Users/jkozlowski/IdeaProjects/daniels-regex-interpreter/target/scala-2.13/classes ...
[success] Total time: 7 s, completed 24 Mar 2021, 17:03:24
[success] Total time: 0 s, completed 24 Mar 2021, 17:03:24
[info] Defining scalaVersion
[info] The new value will be used by Compile / bspBuildTarget, Compile / dependencyTreeCrossProjectId and 56 others.
[info] 	Run `last` for details.
[info] Reapplying settings...
[info] set current project to ce3.g8 (in build file:/Users/jkozlowski/IdeaProjects/daniels-regex-interpreter/)
[success] Total time: 0 s, completed 24 Mar 2021, 17:03:24
[info] We are going to migrate your project root to 3.0.0-RC1
2021.03.24 17:03:27:106 [pool-25...] [INFO ] migrate.Scala3Migrate.previewMigration:29 - Migrating /Users/jkoslowski/IdeaProjects/daniels-regex-interpreter/src/main/scala/com/example/Regular.scala
2021.03.24 17:03:34:921 [pool-25...] [INFO ] migrate.Scala3Migrate.previewMigration:34 - The project compiles successfully in Scala 3
2021.03.24 17:03:34:951 [pool-25...] [INFO ] migrate.Scala3Migrate.compileWithRewrite:109 - Finalizing the migration: compiling in scala 3 with -rewrite option
[info]
[info]
[info] root project has successfully been migrated to scala 3.0.0-RC1
[info] You can now commit the change!
[info] You can also execute the compile command:
[info]
[info] root / compile
[info]
[info]
```

Note that this did change `scalaVersion` in the shell, but that wasn't persisted in the files.

Platform: macOS Catalina 10.15.7

**Comment by mlachkar** [comment]
Thanks for testing the tool.
So scala3-migrate provides 4 commands: 
- two of them, `migrate-libs` and `migrate-scalacOptions` don't change the build, but propose how to change it. 
- migrate-syntax and migrate, modify the code if necessary to make it compile in Scala 3

In your case, your code compiles in scala 3. You can now either change the scalaVersion of your project, or add Scala 3 as a crossScalaVersions. 
```
ThisBuild  /  scalaVersion := scala3
ThisBuild  / crossScalaVersions := List(scala213, scala3)
```
Maybe we can modify the message log to propose that. What do you think ?

**Comment by kubukoz** [comment]
Yeah, I think just suggesting that the user update the build themselves (in consistency with `migrate-libs` and `migrate-scalacOptions` will be good enough :)

**Comment by nafg** [comment]
The problem is that sbt has no well-defined way of modifying dependencies. Scala Steward is pretty amazing in its heuristics; if the code there could be factored out as a general-purpose "sbt-dependencies-updating tool" that would be neat. Of course it would forever have major caveats on it.

**Comment by kubukoz** [comment]
Yeah, it's too complex. Somehow i forgot about the intricacies because usual libraries can be updated :)

**Comment by mlachkar** [comment]
Hello,
We have improved the log message to suggest modifying the scalaVersion or adding a crossScalaVersion.
It should be released in 0.3.4! 
Thanks for testing the tool.
